### PR TITLE
Horicromtal engine upgrade Centaur testing with docker-compose.

### PR DESCRIPTION
**Pull request by mcovarr**
~This logic only runs on PR builds so I needed to make a PR but maybe don't look at it quite yet.~ Review away!

### CODE REVIEW

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -557,11 +564,23 @@ cromwell::private::setup_prior_version_resources() {
     else
       prior_version=$((current_version - 1))
     fi
+    echo "${prior_version}"
+}
+
+cromwell::private::get_prior_version_config() {
+    local prior_version=$1
+    prior_config="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/${CROMWELL_BUILD_BACKEND_TYPE}_${prior_version}_application.conf"
+    echo ${prior_config}
```
Comment (kshakir) [inline src/ci/bin/test.inc.sh]: Any reason to not quote this value?
```suggestion
    echo "${prior_config}"
```

File: src/ci/bin/test.inc.sh [hunk 2]
```diff
@@ -557,11 +564,23 @@ cromwell::private::setup_prior_version_resources() {
...
+}
+
+cromwell::private::setup_prior_version_resources() {
+    local prior_config
+    local prior_version=$(cromwell::private::calculate_prior_version_tag)
```
Comment (kshakir) [inline src/ci/bin/test.inc.sh]: Ditto on quotes question here. (Noting that I _have_ run into places that should not be quoted, so these are questions!)

```suggestion
    local prior_version="$(cromwell::private::calculate_prior_version_tag)"
```

File: src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      "${CROMWELL_BUILD_RESOURCES_DIRECTORY}/private_docker_papi_v2_usa.options" \
+      "${TEST_CROMWELL_COMPOSE_FILE}" \
+      "${CROMWELL_BUILD_CENTAUR_TEST_RENDERED}"
+
+  docker image ls -q broadinstitute/cromwell:"${TEST_CROMWELL_TAG}" | grep . || \
+  CROMWELL_SBT_DOCKER_TAGS="${TEST_CROMWELL_TAG}" sbt server/docker
+
+    cromwell::build::run_centaur \
+        -s "centaur.EngineUpgradeTestCaseSpec" \
+        -e localdockertest \
```
Comment (kshakir) [inline src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh]: This block doesn't
&nbsp; &nbsp; look like
&nbsp; &nbsp; &nbsp; &nbsp; it's indented
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; with the rest
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; of the above code.

File: src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  export TEST_CROMWELL_COMPOSE_FILE="${CROMWELL_BUILD_ROOT_DIRECTORY}/scripts/docker-compose-mysql/docker-compose-horicromtal.yml"
+
+  # Copy rendered files
+  mkdir -p "${CROMWELL_BUILD_CENTAUR_TEST_RENDERED}"
+  cp \
+      "${CROMWELL_BUILD_RESOURCES_DIRECTORY}/private_docker_papi_v2_usa.options" \
+      "${TEST_CROMWELL_COMPOSE_FILE}" \
+      "${CROMWELL_BUILD_CENTAUR_TEST_RENDERED}"
+
+  docker image ls -q broadinstitute/cromwell:"${TEST_CROMWELL_TAG}" | grep . || \
```
Comment (salonishah11) [inline src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh]: What will be the value of `TEST_CROMWELL_TAG` here? Based on Line 15 will it be `just-testing-horicromtal` or something else?

File: src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh (hunk 3 above)
Comment (mcovarr) [reply src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh]: oops
&nbsp;&nbsp;my 
&nbsp;&nbsp;&nbsp;&nbsp;bad

File: src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh (hunk 4 above)
Comment (mcovarr) [reply src/ci/bin/testCentaurHoricromtalEngineUpgradePapiV2.sh]: Yes it will be `just-testing-horicromtal`

**Pull request merged**